### ISSUE TITLE
[Martine] Paired down script proposal

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-people-use-web-videos#24]
The current script has a few things I think wouldn't quite be accurate -- namely using automatic captions for pre-recorded videos (that's usually for live video calls and presentations) and relying on lip reading for a lecture (the school would provide an interpreter or CART typist). So I moved a few things around to help keep things related to an online degree program and (hopefully) accurate to how universities run.

This also reduces the current script by 118 words (possibly more depending on how we want to do the sign-off at the end). The total word count is 243 which is estimated to be just shy of 2 minutes at 130 wpm (see http://speechinminutes.com/).
****************

Scene 1 - Hello! I’m Martine. I was born deaf and the best way I communicate is with sign language - I think and dream in signs. 

_Video: Martine signing directly to us viewers (documentary style into the camera). We briefly see that she is communicating through a sign language interpreter who is speaking what she signs (the audio we hear)._ 

Scene 2 - I recently enrolled in an online degree. For my lectures, I schedule interpreters who sign the conversations and speak aloud my comments. I “pin” their videos to ensure they’re always visible.

_Video: [Martine's interpreter is now a voiceover. Cutaway to a natural setting.] We see Martine in a video conference meeting with multiple people online. We see her pin two sign language interpreters to the main viewing window._ 

Scene 3 - I need interpreters because automatic captions are too hard to follow, especially when it messes up technical phrases, and runs everyone’s words together.

_Video: [Different but similar setting as previous scene.] Martine is in a new lecture (just showing a primary speaker window as in a lunch-and-learn guest speaker) and we see a sentence without punctuation and obviously misspelled words, so that the sentence doesn’t make sense. Martine looks puzzled._ 

Scene 4 - For our assigned videos, however, I do rely on closed captions. These work best when not too long and synced with the audio. It also helps if I can change the text size and colors, and even the position for easier viewing and reading.

_Video: [Several scenes **- this is ambitious**] Martine selects a video with a "CC" icon from a list, watches a well-captioned video, then pauses to explore font size and color options, lastly moving the captions up on the screen and restarting the playback._   

Scene 5 - When video calls are unplanned, then I’ll lip read and speak my responses. I need to see the person’s face and have them speak clearly but it’s not 100% accurate, especially with some accents or several people talking. It’s also tiring so I don’t do it for long periods.

_Video: [New scene] Video call with 2 other people with Martine talking (but we don't hear her). Then we see both attendees speak at the same time._

Scene 6 - Of course school mostly comes with reading. I’m fine with that, except when the writing is complex and isn’t broken up like with headings. Many people don’t realize I’m translating the text I’m reading, so clear and direct is best.

_Video: Perhaps reading an older passage of text that's missing common structure._

Scene 7 - Well, that’s a little about me - making my way through school and getting in some fun when I can! [_Optional:_ I hope this helped you learn how to make the world more accessible.]

_Video: Back to Martine looking at the camera, perhaps on a stool or talking head style._

**Comment by haltersweb** [comment]
I like the personal touch of "I think and dream in signs" though I would say "sign language" rather than "signs"

**Comment by jade-mc** [comment]
Me again, I'm putting a few comments here to keep them together.

For scene 1, I think the additional info on top of 'think and dream' in the original script is needed to strengthen the idea that sign langauge is a native language. 
I also think this info needs to go into scenes 3/4 as another reason why captions aren't fully reliable for some users. In the original, I feel this would need strengthening as well ('the next best option' is in the right place but not strong enough.) 
For both this script and the original, I still feel that the lip reading accuracy issue isn't quite there. Some research into lip reading suggests that only 30% of what we say is communicable by lip reading only, this is really low. I'm worried that the way it's being dealt with in our scripts doesn't quite capture the fact that it's an aid rather than fully reliable. 'not 100% accurate' implies that it's not far off. 
A suggestion: using some lip reading, facial gestures, body language, aids commmunication, so it's useful to be able to see a person's whole face, but on the whole, lip reading isn't fully reliable.

**Comment by haltersweb** [comment]
I really like the personality and personal experience coming out of this script.  It has a feel that draws me in thru both empathy and sympathy.

**Comment by haltersweb** [comment]
Scene 6.  Mention something about spoken/written language has different grammar structure.  Mention that she is translating to "sign language".  Such as : "I’m translating the text I’m reading into sign language".

**Comment by haltersweb** [comment]
I think the decision must be made as to which "voice" we are using.  If we choose to use 1st person voice as in here, we must do so for all.

**Comment by nitedog** [comment]
Addressed in latest revision.
